package.json

```diff
@@ -9,15 +9,25 @@
     "dev": "npm run heroku-postbuild && npm run start",
     "heroku-postbuild": "cd client && npm run build",
     "test": "jest",
-    "posttest": "cd client && npm run build && npm test"
+    "posttest": "cd client && npm run build && npm test",
+    "lint": "./node_modules/.bin/eslint src",
```

Comment:
This should work even without ./node_modules/.bin/ I think. Can you try? http://2ality.com/2016/01/locally-installed-npm-executables.html